services/orion-decision/src/orion_decision/ci_check.py

```diff
@@ -85,5 +85,5 @@ def check_matrix(args):
 
                 # create CIMatrix for each branch and is_release=True
                 for branch in branches:
-                    CIMatrix(matrix, branch, is_release=False)
-                CIMatrix(matrix, None, is_release=True)
+                    CIMatrix(matrix, branch, event_data["type"])
```

Comment:
This isn't right. The event type here will be in `event_data["action"]` and will be one of `github-pull-request`, `github-push`, or `github-release` ... so it requires a lookup like the one in `GithubEvent.from_taskcluster()`